hazelcast/src/main/java/com/hazelcast/config/XmlConfigBuilder.java

```diff
@@ -2107,6 +2107,14 @@ private void handleSecurityPermissions(Node node) throws Exception {
                 type = PermissionType.TRANSACTION;
             } else if ("all-permissions".equals(nodeName)) {
                 type = PermissionType.ALL;
+            } else if ("durable-executor-service-permission".equals(nodeName)) {
```

Comment:
Apart from `cache-permission`, the rest of elements in these conditions were already part of the config XSD but not handled in `XmlConfigBuilder`